get-started/index.md

```diff
@@ -59,10 +62,355 @@ Click "Install" and wait for the installation to complete.
+    ...
+    spec:
+      containers:
+        - name: sample-app
+          image: docker.io/myorg/myapp:latest
+          env:
+            - name: CRYOSTAT_AGENT_APP_NAME
+              # Replace this with any value you like to use to identify your application.
+              value: "myapp"
+            - name: NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: CRYOSTAT_AGENT_BASEURI
+              # Update this to correspond to the name of your Cryostat instance
+              # if it is not 'cryostat'. This assumes that the target application
+              # and the Cryostat instance are in the same Namespace, but you may
+              # choose to configure the Agent to communicate with a Cryostat in
+              # a different Namespace, too.
+              # (https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/)
+              value: https://cryostat.$(NAMESPACE).svc:8181
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+            - name: CRYOSTAT_AGENT_CALLBACK
+              # This infers the Agent Callback directly from the Pod's IP address using the
+              # Kubernetes Downward API. Use this value directly as provided. The port number
+              # 9977 can be changed but must match the containerPort below.
+              value: "http://$(POD_IP):9977"
+              # Replace "abcd1234" with a base64-encoded authentication token. For example,
+              # in your terminal, do 'oc whoami -t | base64' to use your user account's
+              # token as the token that the Agent will pass to authorize itself with
+              # the Cryostat server.
+            - name: CRYOSTAT_AGENT_AUTHORIZATION
+              value: "Bearer abcd1234"
+              # This environment variable is key to the "hybrid" setup.
+              # This instructs the Agent to register itself with Cryostat
+              # as reachable via JMX, rather than reachable via HTTP.
+            - name: CRYOSTAT_AGENT_REGISTRATION_PREFER_JMX
+              value: "true"
+              # Here we configure the application to load the Agent JAR as
+              # well as to enable JMX, since we want the Agent to register
+              # itself as reachable via JMX.
+            - name: JAVA_OPTS
+              value: >-
+                -javaagent:/deployments/app/cryostat-agent.jar
+                -Dcom.sun.management.jmxremote.port=9091
+                -Dcom.sun.management.jmxremote.ssl=false
+                -Dcom.sun.management.jmxremote.authenticate=false
+          ports:
+            - containerPort: 9977
+              protocol: TCP
+          resources: {}
+      restartPolicy: Always
+status: {}
+```
+
+Create an application `Service`:
+```yaml
+apiVersion: v1
+kind: Service
+...
+spec:
+  ports:
+    - name: "jfr-jmx"
+      port: 9091
+      targetPort: 9091
```

Comment:
I think we might be able to skip this part, since the discovery will be from the agent?